docs/manage/CLI.md

```diff
@@ -95,7 +82,8 @@ echo 'export PATH="$HOME/go/bin:$PATH"' >> ~/.bashrc
 {{% /tab %}}
 {{< /tabs >}}
 
-To later update the Viam CLI tool, you can use the steps above to reinstall the latest version.
+To later update the Viam CLI tool on Linux, use the steps above to reinstall the latest version.
+to later update the Viam CLI tool on macOS, run `brew upgrade viamrobotics/brews/viam`.
```

Comment:
if they went through `brew tap` above, can also do `brew upgrade viam`, but this works too

Reply:
Good point!